.github/workflows/CI.yml

```diff
@@ -51,11 +51,11 @@ jobs:
         - { os: ubuntu-22.04  , ocaml-version: 4.14.0 }
         - { os: ubuntu-20.04  , ocaml-version: 4.14.0 }
         - { os: ubuntu-18.04  , ocaml-version: 4.14.0           , publish: true }
+        - { os: ubuntu-18.04  , ocaml-version: "4.14.0+options,ocaml-option-musl,ocaml-option-static,ocaml-option-flambda"  , publish: true }
```

Comment:
So we are going to ride 18.04 to the bitter end for main builds.  I'm ok with that.